doc/security.md

```diff
@@ -112,3 +122,15 @@ trusted.
 
 This happens if another trusted client or the local server administrator
 removed the trust entry on the server.
+
+
+# Production setup
+For production setup, it's reccomended that `core.trust_password` is unset
+after all clients have been added.
+
+This prevents brute-force attacks trying to guess the password.
```

Comment:
I guess this is meant as an addendum to the above sentence about unsetting trust password? :) If so, then remove the newline and make it one paragraph. :)